libs/wingsdk/test/target-sim/__snapshots__/file-counter.test.ts.snap

```diff
@@ -17,20 +17,60 @@ return class Handler {
         }
 };
 })())({
-counter: (function(env) {
-  let handle = process.env[env];
+counter: (function() {
+  const handle = process.env[\\"COUNTER_HANDLE_4ecd8d46\\"];
   if (!handle) {
     throw new Error(\\"Missing environment variable: \\" + env);
   }
-  return $simulator.findInstance(handle);
-})(\\"COUNTER_HANDLE_4ecd8d46\\"),
-bucket: (function(env) {
-  let handle = process.env[env];
+  const simulatorUrl = process.env.WING_SIMULATOR_URL;
+  if (!simulatorUrl) {
+    throw new Error(\\"Missing environment variable: WING_SIMULATOR_URL\\");
+  }
+
+  return new Proxy({}, {
+    get: function(target, method, receiver) {
+      return async function() {
+        const resp = await fetch(simulatorUrl + \\"/v1/call\\", {
+          method: \\"POST\\",
+          headers: { \\"Content-Type\\": \\"application/json\\" },
+          body: JSON.stringify({ handle, method, args: Array.from(arguments) }),
+        });
+        const parsed = await resp.json();
+        if (parsed.error) {
+          throw new Error(parsed.error);
+        }
+        return parsed.result;
+      };
+    }
+  });
+})(),
```

Comment:
Can this generated code do something like the example below?

```ts
import { createSimulatorProxy } from "@winglang/sdk";

// ...
counter: createSimulatorProxy(process.env.WING_SIMULATOR_URL),
// ...
```

Reasoning is smaller bundles, and also due to possibly importing superjson (as a dependency from @winglang/sdk). It's reasonable to require having `@winglang/sdk` installed and available inside the astro project.

Reply:
Added something like this (see docs). Totally experimental of course